src/Tizen.Sensor/Tizen.Sensor/Sensor.cs

```diff
@@ -545,5 +652,606 @@ private void DestroyHandles()
         {
             Interop.SensorListener.DestroyListener(_listenerHandle);
         }
+
+        /// <summary>
+        /// Gets the handle for the default sensor of a given sensor URI.
+        /// </summary>
+        /// <param name="uri">
+        /// A sensor or a sensor type URI to get the handle of its default sensor.
+        /// </param>
+        /// <param name="sensorHandle">
+        /// The default sensor handle.
+        /// </param>
+        /// <since_tizen> 6 </since_tizen>
+        public void GetDefaultSensorByUri(string uri, out IntPtr sensorHandle)
```

Comment:
Do not use IntPtr and out param in public API

Reply:
Inptr is removed from all public API's